Sources/SwiftBuild/SWBBuildServiceSession.swift

```diff
@@ -12,8 +12,8 @@
 
 import Foundation
 
-import SWBProtocol
-import SWBUtil
+public import SWBProtocol
```

Comment:
Generally we've tried not to expose SWBProtocol and SWBUtil as public imports, though here it might require duplicating some of the response types

Reply:
I think it's a hard requirement that we don't expose them.